src/components/PageHeader/PageHeader.tsx

```diff
@@ -122,7 +121,10 @@ const PageHeader = () => {
               <button
                 className="new-measure"
                 data-testid="create-new-cql-library-button"
-                onClick={() => history.push("/cql-libraries/create")}
+                onClick={() => {
```

Comment:
className="measures"
className="new-measure"
Can we update these class names and use a generic name? since it is being reused and misleading.

Reply:
Definitely, updated to 'create-button'